File: LeetCode_3688/Main.php
<?php

class Solution {

    /**
     * @param Integer[] $nums
     * @return Integer
     */
    function evenNumberBitwiseORs($nums) {
        $answer = 0;
        foreach ($nums as $num) {
            $answer = ($num & 1 ? $answer : $answer | $num);
        }
        return $answer;
    }
}


?>

File: LeetCode_3688/Main.cpp
#include<bits/stdc++.h>
using namespace std;

class Solution {
public:
    int evenNumberBitwiseORs(vector<int>& nums) {
        int answer = 0;
        for (int num : nums) {
            answer = (num & 1 ? answer : answer | num);
        }
        return answer;
    }
};